jupyterhub/utils.py

```diff
@@ -288,13 +288,8 @@ async def is_reachable():
             else:
                 app_log.debug("Server at %s responded with %s", url, e.code)
                 return e.response
-        except OSError as e:
-            if e.errno not in {
-                errno.ECONNABORTED,
-                errno.ECONNREFUSED,
-                errno.ECONNRESET,
-            }:
-                app_log.warning("Failed to connect to %s (%s)", url, e)
```

Comment:
Sorry, I wasn't clear what `except Exception` should replace. This OSError block should stay. We still want to avoid logging the _expected_ "not there yet" errors, but log and not fail on unexpected errors.

Reply:
Ah sure. Sorry for that. Should be fine now.